__tests__/endpoints/app.test.js

```diff
@@ -80,6 +80,7 @@ describe("GET /api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comments_count: 11,
```

Comment:
Generally if possible it would be better to not have to go back and alter previous tests. Because you've used toMatchObject, I think your test would still pass without the comments_count in it. I would advise separate tests for new tickets, even if the new test is just checking this one property 😄